pkg/cmd/setup.go

```diff
@@ -66,6 +73,11 @@ func setup(cmd *cobra.Command, args []string) {
 		utils.Log(valueFromEnvironmentNotice("DOPPLER_PROJECT"))
 		selectedProject = localConfig.EnclaveProject.Value
 	default:
+		if repoConfig.Setup.Project != "" {
+			utils.Log("Reading project from doppler.yaml")
```

Comment:
After testing locally, I think this message should more closely match our error message "Unable to parse doppler repo config file". Otherwise it's not clear to me that these are referring to the same thing.
```suggestion
			utils.Log("Reading project from repo config file doppler.yaml")
```